content/tutorials/portenta-h7/por-ard-lvgl2/content.md

```diff
@@ -354,7 +354,79 @@ To finish with the functions, the mouse pointer for the LVGL mouse driver, save
 
 Once we have added these functions, our interface will be created. Then, we need to go back to the main tab of the `lvgl_interaction.ino` sketch to finish the programming of the interface.
 
-## 4. Add the Mouse and Keyboard receivers
+## 4. Create the Mouse and Keyboard RPC senders
+
+Following our structure, the **Portenta M4** will handle the USB inputs (mouse and keyboard) and send the required data to the **Portenta M7** using **RPC**
+
+Create a new sketch called *lvgl_mouse_m4.ino*.
+
+The sketch will contain the next functions
+* Mouse
+    * `mouseCallback(tusbh_ep_info_t* ep, const uint8_t* mouse)`  
+        This function handle the mouse buttons and the relative change of position
+* Keyboard
+    * `keyboardCallback(tusbh_ep_info_t* ep, const uint8_t* keys)`  
+        Get the input keys, proccess them and send the correct character.
+
+**Note**: This functions are defined in the `Portenta_USBhost.h` wrapper, because it needs to be defined for the USB API structure of the USB library, defining them its needed to create the logic of those events.
+
+### Include needed libraries
```

Comment:
Remove the subheading from here. We don't have subheading inside Steps

Reply:
```suggestion
**Include needed libraries**
```